volatility3/framework/plugins/windows/psxview.py

```diff
@@ -114,10 +110,10 @@ def _check_pslist(self, tasks):
         return self._proc_list_to_dict(tasks)
 
     def _check_psscan(
-        self, layer_name: str, symbol_table: str
+        self,
     ) -> Dict[int, extensions.EPROCESS]:
         res = psscan.PsScan.scan_processes(
-            context=self.context, layer_name=layer_name, symbol_table=symbol_table
+            context=self.context, kernel_module_name=self.config["kernel"]
```

Comment:
Hmmm, this is ok, but will make it harder to turn this into a classmethod if we ever want in the future.  If we were going to do that we'd need to take in context as well, so probably not worth changing, just something to keep in mind.  I think this is one of the very few plugins left that have actual non-classmethod methods...

Reply:
Yea, I didn't want to rewrite psxview so extensively as its essentially calling the proces enumeration functions anyway then doing whatever conversion is needed to get a EPROCESS from it. Making psxview have classmethods for that would duplicate what the plugins are providing anyway, so keeping it how it is seems best. And yes, I am happy with the other changes and they have been tested with running all the plugins over many samples.